api/proto/node/node.go

```diff
@@ -186,3 +194,20 @@ func DeserializeEpochShardStateFromMessage(payload []byte) (*types.EpochShardSta
 
 	return epochShardState, nil
 }
+
+// ConstructCXReceiptsMessage constructs cross shard receipts and merkle proof
+func ConstructCXReceiptsMessage(cxs types.CXReceipts, mkp *types.CXMerkleProof) []byte {
+	msg := &CXReceiptsMessage{CXS: cxs, MKP: mkp}
+
+	byteBuffer := bytes.NewBuffer([]byte{byte(proto.Node)})
+	byteBuffer.WriteByte(byte(Block))
+	byteBuffer.WriteByte(byte(Receipt))
+	by, err := rlp.EncodeToBytes(msg)
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Don't use fatal (don't exit if not necessary)

Reply:
changed